Reference/Management/Models/DataType-v9.md

```diff
@@ -0,0 +1,75 @@
+---
+versionFrom: 9.0.0
+verified-against: rc001
+meta-title: DataType Model
+meta.Description: A DataType is what you see in the backoffice in the Settings / DataTypes tree. The listed nodes are definitions of the DataTypes that are available to use on your PropertyTypes.
+---
+
+# DataType
+
+A DataType is what you see in the backoffice in the Settings / DataTypes tree. The listed nodes are definitions of the DataTypes that are available to use on your PropertyTypes.
+
+* **Namespace:** `Umbraco.Cms.Core.Models`
+* **Assembly:** `Umbraco.Core.dll`
+
+All samples in this document will require references to the following dll:
+
+* Umbraco.Core.dll
+
+All samples in this document will require the following using statement:
+
+```csharp
+using Umbraco.Cms.Core.Models;
+```
+
+## Constructors
+
+### new DataType(IDataEditor editor, IConfigurationEditorJsonSerializer serializer, int parentId = -1)
+
+Constructor for creating a new `DataType` object where the necessary parameters are a `IDataEditor` and a `IConfigurationEditorJsonSerializer`.  Optionally, the parentId can be added, if not provided the default value is -1, which means it will be created at root level.
```

Comment:
```suggestion
Constructor for creating a new `DataType` object where the necessary parameters are a `IDataEditor` and a `IConfigurationEditorJsonSerializer`. Optionally, the parentId can be added, if not provided the default value is -1, which means it will be created at root level.
```

Reply:
@busrasengul Updated 😄